.shippable.yml

```diff
@@ -20,11 +20,14 @@ build:
     - function upload_cache() { if [ ! -e .uploaded ]; then echo Uploading cache && tar c -C /root .ccache | gzip -1 | ssh $SCP_OPT shippable@shippable-cache.forissier.org "cat >ccache-$PROJ.tar.gz" && touch .uploaded || echo Nevermind; fi; }
     - function check_upload_cache() { NOW=$(date +%s); if [ $(expr $NOW - $START) -gt 3000 ]; then upload_cache; fi; }
     - function _make() { make -j$(getconf _NPROCESSORS_ONLN) -s O=out $* && ccache -s && ccache -z && check_upload_cache; }
+    - function download_plug_and_trust() { curl -L https://github.com/foundriesio/plug-and-trust/releases/download/v0.0.1/se050-0.0.1.tar.bz2 | tar -jxvf - -C /root || (rm -rf /root/se050-0.0.1; echo Nervermind); }
 
     - download_cache
     - ccache -z
+    - download_plug_and_trust
 
     - _make
+    - if [ -d /root/se050-0.0.1 ]; then _make PLATFORM=imx-mx8mmevk CFG_CRYPTO_DRV_CIPHER=y CFG_CRYPTO_DRV_ACIPHER=y CFG_IMX_I2C=y CFG_NXP_SE05X=y CFG_NXP_CAAM=n CFG_NXP_SE05X_RNG_DRV=y CFG_WITH_SOFTWARE_PRNG=n CFG_NXP_SE05X_RSA_DRV=y CFG_NXP_SE05X_ECC_DRV=y CFG_NXP_SE05X_CTR_DRV=y CFG_NXP_SE05X_PLUG_AND_TRUST_LIB=/root/se050-0.0.1/build/libse050.a CFG_NXP_SE05X_PLUG_AND_TRUST=/root/se050-0.0.1 CFG_STACK_THREAD_EXTRA=8192 CFG_STACK_TMP_EXTRA=8192 ; fi
```

Comment:
Would you mind moving this one after the line: `_make PLATFORM=imx-mx8mmevk CFG_NXP_CAAM=y`?

Reply:
sure